src/sentry/utils/compat/__init__.py

```diff
@@ -34,58 +34,34 @@ def _identity(x):
     return x
 
 
-if six.PY2:
-    # https://github.com/pallets/werkzeug/blob/master/werkzeug/_compat.py
-    def implements_to_string(cls):
-        cls.__unicode__ = cls.__str__
-        cls.__str__ = lambda x: x.__unicode__().encode("utf-8")
-        return cls
-
-    def implements_iterator(cls):
-        cls.next = cls.__next__
-        del cls.__next__
-        return cls
-
-    def implements_bool(cls):
-        cls.__nonzero__ = cls.__bool__
-        del cls.__bool__
-        return cls
-
-    from binascii import crc32
-
-    import imp
-
-    def new_module(name):
-        return imp.new_module(name)
-
-
-else:
-    implements_to_string = _identity
-    implements_iterator = _identity
-    implements_bool = _identity
-
-    from binascii import crc32 as _crc32
-
-    # In python3 crc32 was changed to never return a signed value, which is
-    # different from the python2 implementation. As noted in
-    # https://docs.python.org/3/library/binascii.html#binascii.crc32
-    #
-    # Note the documentation suggests the following:
-    #
-    # > Changed in version 3.0: The result is always unsigned. To generate the
-    # > same numeric value across all Python versions and platforms, use
-    # > crc32(data) & 0xffffffff.
-    #
-    # However this will not work when transitioning between versions, as the
-    # value MUST match what was generated in python 2.
-    #
-    # We can sign the return value using the following bit math to ensure we
-    # match the python2 output of crc32.
-    def crc32(*args):
-        rt = _crc32(*args)
-        return rt - ((rt & 0x80000000) << 1)
-
-    import types
-
-    def new_module(name):
-        return types.ModuleType(name)
+implements_to_string = _identity
+implements_iterator = _identity
+implements_bool = _identity
+
+from binascii import crc32 as _crc32
+
+# In python3 crc32 was changed to never return a signed value, which is
+# different from the python2 implementation. As noted in
+# https://docs.python.org/3/library/binascii.html#binascii.crc32
+#
+# Note the documentation suggests the following:
+#
+# > Changed in version 3.0: The result is always unsigned. To generate the
+# > same numeric value across all Python versions and platforms, use
+# > crc32(data) & 0xffffffff.
+#
+# However this will not work when transitioning between versions, as the
+# value MUST match what was generated in python 2.
+#
+# We can sign the return value using the following bit math to ensure we
+# match the python2 output of crc32.
```

Comment:
Would be good to update this comment now to reflect why this is now here for historical reasons.